RecoEgamma/EgammaMCTools/python/pfClusterMatchedToPhotonsSelector_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+particleFlowClusterECALMatchedToPhotons = cms.EDProducer('PFClusterMatchedToPhotonsSelector',
```

Comment:
I will remove the description here and leave the python file, if that's ok with you.